report.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sbn
+import json
+
+
+class ReportAssistant():
+    def __init__(self):
+        self.usage = pd.read_csv("./state/metrics/usage.csv")
+        # self.messages = pd.read_csv("./state/metrics/messages.csv")
+        self.feedback = pd.read_csv("./state/metrics/feedback.csv")
+        self.current_df = None
+
+    def read_df(self, df: pd.DataFrame, n=10):
+        print(df.head(n))
+        return df
+
+    def view_graph(self, df: pd.DataFrame, type = 'bar chart'):
+        df = df[["user_id", "output_tokens"]]
+        if type == "bar chart":
+            print(f'You chose a bar chart to view {self.current_df}')
+            print(f"{df.groupby(['user_id']).output_tokens.sum().reset_index()}")
```

Comment:
Debugging?

If there is information we want in the logs, we should use the `logging` module, as we do in other parts of the duck bot.

Reply:
Yes debugging, but I didn't think about the logs. I'll start adding that where necessary.